src/components/TransactionsGroup/FiltersModal/FiltersModal.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { Pressable, Text, View, TextInput } from 'react-native';
+import styles from './styles';
+import globalStyles from '../../../globalStyles';
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+import SvgIcon from '../../../../assets/SvgIcon';
+
+
+const FiltersModal = ({ refRBSheet, minDate, setMinDate, showMin, setShowMin, maxDate, setMaxDate, showMax, setShowMax, minAmount, 
+    setMinAmount, maxAmount, setMaxAmount, typeFilter, setTypeFilter, handleReset, handleApply }: any) => {
+  return (
+    <>
+        <Pressable style={styles.modalIconContainer} onPress={() => refRBSheet.current.close()}>
+            <Text style={styles.closeText}>Close</Text>
+        </Pressable>
+        <Text style={styles.modalTitle}>Filter Transactions</Text>
+        <View style={styles.innerModal}>
+            <View style={styles.filtersContainer}>
+                <View style={styles.filterGroup}>
+                    <View style={styles.withClear}>
+                        <Text style={styles.filterBtnText}>Filter by date</Text>
+                        <Text style={[styles.closeText, ((minDate !== null && maxDate !== null) ? { color: '#526DC2' } : {color: '#A9A9A9'})]}
+                            onPress={() => {
+                                setMinDate(null)
+                                setMaxDate(null)
+                            }}
+                        >
+                            Clear
+                        </Text>
+                    </View>
+                    <View style={styles.amountsContainer}>
+                        <Pressable onPress={() => setShowMin(true)} style={[styles.dateInput, (minDate !== null) ? styles.selectedInput : styles.unselectedInput]}>
```

Comment:
Your code always looks good so I have to resort to nitpicks.
For long lines like these let's move style to the next line so the one line's not as long